src/fu-self-test.c

```diff
@@ -6947,6 +6948,7 @@ fu_remote_auth_func(void)
 		"  \"FirmwareBaseUri\" : \"https://my.fancy.cdn/\",\n"
 		"  \"Username\" : \"user\",\n"
 		"  \"Password\" : \"pass\",\n"
+		"  \"AuthType\" : \"none\",\n"
```

Comment:
i don't think we want to include "none" -- if it's none, just don't include it in the output.